src/NBomber/Infra/Reporting.fs

```diff
@@ -11,20 +11,31 @@ open NBomber.Contracts
 open NBomber.Domain
 open NBomber.Statistics
 
-let buildReport (scenario: Scenario, flowInfo: FlowInfo[]) =        
+let buildReport (scenario: Scenario, flowInfo: FlowInfo[]) = 
+    let getPausedTime (step: Step) =
+        match step with
+        | Pause time -> time.Ticks
+        | _ -> int64 0
+
     let envInfo = HostEnvironmentInfo.getEnvironmentInfo()   
     let header  = printScenarioHeader(scenario)
     
+    let activeStepsDuration = scenario.TestFlows
```

Comment:
can you please wrap this in local function getPauseTime(scenario)
then you should get very simple interaction:
let actualTime = scenario.Time - getPauseTime(scenario)